ChromeExtension/test.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>  
```

Comment:
If this is just for testing, does it need to be checked in?

Reply:
Yeah, I guess I can delete it. Thanks. It is just a habit to push all my files and not pick and choose